integration/helpers/yaml_utils.py

```diff
@@ -17,7 +19,7 @@ def load_yaml(file_path):
     """
     with open(file_path) as f:
         data = f.read()
-    return yaml.load(data, Loader=yaml.FullLoader)
+    return yaml_parse(data)
```

Comment:
Not sure why this is showing as modified as the code on develop is already like this.